openg2p_registration/models/openg2p_registration.py

```diff
@@ -346,17 +369,83 @@ def create_beneficiary_from_registration(self):
             'context': context
         }
 
+    @api.onchange('stage_id')
+    def checking_duplicates(self):
+        if int(self.stage_id.id) == 2:
+            r = self.search_beneficiary()
+            if r == 200:
+                # Add beneficiary.id to duplicate_beneficiaries_ids
+                pass
+            else:
+                self.index_beneficiary()
```

Comment:
Indexing needs to be in the create_beneficiary_from_registration as discussed yesterday.